### ISSUE TITLE
Error: undefined is not an object (evaluating 'cordova.plugins.firebase')

### ISSUE THREAD

**Issue by willnix86** [chemerisuk/cordova-plugin-firebase-messaging#96]
I receive the above error on both Android and iOS. Any reference to cordova.plugins.firebase happens after the 'Device ready' event.

**Comment by willnix86** [comment]
Same issue as raised [here](https://github.com/chemerisuk/cordova-plugin-firebase-analytics/issues/105)

**Comment by fujaru** [comment]
May be late to this but I was also having this problem. Apparently for me, the problem is that I was using cordova-ios version 5.1.1.
I solved this by updating cordova-ios to 6.1.1 and reinstalling cordova-plugin-firebase-messaging.
```
cordova platform remove ios
cordova platform add ios
cordova plugin remove cordova-plugin-firebase-messaging
cordova plugin add cordova-plugin-firebase-messaging
```
